scripts/qa_crowbarsetup.sh

```diff
@@ -2294,6 +2311,16 @@ function hacloud_configure_cluster_members()
                 "['attributes']['pacemaker']['stonith']['per_node']['nodes']['$node']['params']" "''"
         done
     fi
+
+    if [ -n "$want_sbd" ] ; then
```

Comment:
this is non-zero length bash check, son `want_sbd=0` will return true, isn't? Not consistent with L#2049 check